src/Microsoft.AspNetCore.Blazor.Browser/Interop/TrackedReference.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.Blazor.Browser.Interop
+{
+    internal class TrackedReference
```

Comment:
This doesn't need to be an instantiatable class, because no other code reads the `.Id` property. But in any case hopefully it can be eliminated entirely in favour of just tracking `TaskCompletionSource` instances.